lib/Refund.php

```diff
@@ -32,29 +32,14 @@ class Refund extends ApiResource
     use ApiOperations\Retrieve;
     use ApiOperations\Update;
 
-    /**
-     * Possible string representations of the failure reason.
-     *
-     * @see https://stripe.com/docs/api/refunds/object#refund_object-failure_reason
-     */
-    const FAILURE_REASON = 'expired_or_canceled_card';
+    const FAILURE_REASON_EXPIRED_OR_CANCELED_CARD = 'expired_or_canceled_card';
```

Comment:
This one is technically a breaking change, but the old constant had a blatantly wrong name :/ `FAILURE_REASON` instead of `FAILURE_REASON_EXPIRED_OR_CANCELED_CARD`.

We can use an override to keep the old value around temporarily if needed.

Reply:
Yeah I'd do an override to play it safe, even if the name was horrible. But I'm fine with risking it